modules/http4s/test/src-js/smithy4s/compliancetests/EnvCompat.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ *  Copyright 2021-2022 Disney Streaming
+ *
+ *  Licensed under the Tomorrow Open Source Technology License, Version 1.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     https://disneystreaming.github.io/TOST-1.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package smithy4s.compliancetests
+
+import scala.scalajs.js
+trait EnvCompat {
```

Comment:
Good to see you building an intuition for stuff like this (ie: what's gonna be platform-specific and how to implement such things), but CE already [provides it](https://github.com/typelevel/cats-effect/blob/9aade3c348b847d9c250b0e2028c691d37cffa09/std/shared/src/main/scala/cats/effect/std/Env.scala#L51)